odxtools/comparam_subset.py

```diff
@@ -108,6 +108,7 @@ def __init_from_et__(self, et_element, doc_frags: List[OdxDocFragment]) -> None:
             if cp_el.tag in ("COMPARAM", "COMPLEX-COMPARAM"):
                 self.comparams.append(create_any_comparam_from_et(cp_el, doc_frags))
 
+        self.complex_physical_default_value = None
```

Comment:
Why?

Reply:
without this, it is possible that a comparam does not exhibit a `complex_physical_default_value` attribute, i.e., 
```python
print(comparam.complex_physical_default_value)
```
will raise an `AttributeError` instead of printing `None`...

Reply:
They need to put a fine on "driving and fixing bug", same way there is a fine on "driving and texting" 😛

Reply:
Right, I suppose that we should create a "drive-by fix jar" instead of a "swear jar". If there ever will be an odxtools-developer meeting, this basically will cover all your bases :D